src/bsoncxx/json.cpp

```diff
@@ -89,5 +89,9 @@ document::value BSONCXX_CALL from_json(stdx::string_view json) {
     return document::value{buf, length, bson_free_deleter};
 }
 
+document::value BSONCXX_CALL operator"" _bson(const char* str, size_t) {
+    return from_json(str);
```

Comment:
`str` is a NULL terminated C string.
`from_json` takes a `string_view`.
Because `string_view` is constructed without a length, it'll end up computing the length of the string (constructor 4 here: https://en.cppreference.com/w/cpp/string/basic_string_view/basic_string_view)
I suggest explicitly constructing a `string_view`, and using the `size_t` argument passed to call constructor 3 of `string_view`. That'll avoid an extra length calculation. It's a small optimization, but also a simple one to make.